deploy/cdk_exec_policy/cdkExecPolicy.yaml

```diff
@@ -0,0 +1,180 @@
+AWSTemplateFormatVersion: 2010-09-09
```

Comment:
I do not see any permissions related to tags. When you created datasets and environments using this execution role, did the resources get tagged?